### ISSUE TITLE
Why `Ctrl+Enter` instead of `Shift-Enter` to run a cell?

### ISSUE THREAD

**Issue by fzumstein** [microsoft/python-in-excel#29]
Jupyter notebook uses `Shift-Enter` so I am hitting that all the time before remembering that it is `Ctrl-Enter` with Python in Excel...

**Comment by keyur32** [comment]
Thank you for the report. Marking as enhancement for the team to take under consideration.

**Comment by keyur32** [comment]
One note is Shift+Enter was already taken by Excel formulas. It will commit the formula and move the cell up one row.  

That said, we'll continue to look to see if there's folks working in Python that might be prefer Shift+enter instead of ctrl+enter while in Beta.

**Comment by ncalm** [comment]
Shift+Enter is "Run cell and create new or select cell below" in Jupyter notebooks and Ctrl+Enter is "Run cell". 

For me at least, the latter and current implementation is more natural for "Run Python cell" in Excel.

**Comment by fzumstein** [comment]
Fair point, so `Shift+Enter` should be introduced as an *additional* shortcut rather than replacing `Ctrl-Enter` to run the Excel cell and move the cursor to the next Python cell.

**Comment by SergeiStPete** [comment]
Just in case, in Power Pivot formula bar Shift+Enter means new line. For Excel data modellers it could be confusing to use it another way.

**Comment by fzumstein** [comment]
> One note is Shift+Enter was already taken by Excel formulas. It will commit the formula and move the cell up one row.

The counter argument to this is that `Enter` also reacts differently in a PY cell than with Excel formulas...
